src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  color: #000;
+  font-family: Roboto, sans-serif;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+}
+
+.list__item {
+  display: flex;
+  align-items: stretch;
+  margin: 0 10px;
+}
+
+.nav__link {
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  cursor: pointer;
+  font-weight: 500;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Create css variable for values that appears multiple times, is this case you can create variable for this blue color `#00acdc;`